src/igvfd/audit/in_vitro_system.py

```diff
@@ -13,64 +14,67 @@ def audit_targeted_sample_term_check(value, system):
     '''
     [
         {
-            "audit_description": "In vitro systems are not expected to have the same targeted_sample_term and sample_terms.",
-            "audit_category": "inconsistent targeted_sample_term",
+            "audit_description": "In vitro systems are expected to have a targeted sample term that is distinct from the starting sample term.",
+            "audit_category": "inconsistent targeted sample term",
```

Comment:
I think that since the property name is "targeted sample term" it's fine. And "inconsistent targeted sample" might be misleading since that sounds like a reference to a sample object